inc/Engine/WPRocketUninstall.php

```diff
@@ -80,6 +80,8 @@ class WPRocketUninstall {
 		'wp_rocket_pricing_timeout',
 		'wp_rocket_pricing_timeout_active',
 		'rocket_get_refreshed_fragments_cache',
+		'user_information_timeout_active',
```

Comment:
Transients to be added to the Test_Uninstall test

Reply:
I've added it.